oak_functions/loader/src/server.rs

```diff
@@ -645,3 +647,11 @@ fn map_host_errors(
         |_| OakStatus::Ok as i32,
     ))))
 }
+
+/// Converts a binary sequence to a string if it is a valid UTF-8 string, or formats it as a numeric
+/// vector of bytes otherwise.
+pub fn try_from_utf8(v: &[u8]) -> String {
```

Comment:
```suggestion
pub fn from_utf8(v: &[u8]) -> String {
```
Since conventionally `try_from` returns `Result`.

Reply:
Renamed to `format_bytes`